prediction_market_agent/agents/microchain_agent/messages_functions.py

```diff
@@ -80,7 +79,7 @@ def __call__(self) -> BlockchainMessage | None:
         # ToDo - Fund the treasury with xDai.
         if not message_to_process:
             logger.info("No messages to process.")
-        return message_to_process
+        return str(message_to_process) if message_to_process else ""
```

Comment:
Returning "" as message instead of None (all other functions also return str, never None)

Reply:
str is fine, but instead of `""`, nicer would be something like "No new messages.", to let LLM know what's going on